From 310fd52e9ec0518d1be1fd9ead3bae6aa74c9408 Mon Sep 17 00:00:00 2001
From: reichi <reichi@opendreambox.org>
Date: Mon, 22 Sep 2014 20:03:13 +0200
Subject: [PATCH] SoftwareManager: fix restore

---
 usr/lib/enigma2/python/Plugins/SystemPlugins/SoftwareManager/plugin.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/lib/enigma2/python/Plugins/SystemPlugins/SoftwareManager/plugin.py b/usr/lib/enigma2/python/Plugins/SystemPlugins/SoftwareManager/plugin.py
index d8f2356..8c279cf 100644
--- a/usr/lib/enigma2/python/Plugins/SystemPlugins/SoftwareManager/plugin.py
+++ b/usr/lib/enigma2/python/Plugins/SystemPlugins/SoftwareManager/plugin.py
@@ -284,7 +284,7 @@ class UpdatePluginMenu(Screen):
 			self.session.open(MessageBox, _("Backup failed."), MessageBox.TYPE_INFO, timeout = 10)
 
 	def startRestore(self, ret = False):
-		if (ret == True):
+		if ret:
 			self.exe = True
 			self.session.open(RestoreScreen, runRestore = True)
 
-- 
1.9.1

